### ISSUE TITLE
empty-block report about empty functions

### ISSUE THREAD

**Issue by ikokostya** [palantir/tslint#2107]
### Bug Report

- __TSLint version__: `4.3.1`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__:  CLI

#### TypeScript code being linted

```ts
function f() {}
```

with `tslint.json` configuration:

```json
{
   "no-empty": true
}
```

#### Actual behavior

Error: `block is empty`.

#### Expected behavior

No error (like in eslint). Eslint has separate rule for this case http://eslint.org/docs/rules/no-empty-function, which also covers empty constructors.

**Comment by adidahiya** [comment]
we could take a PR for a rule option to the existing rule to `allow-empty-functions`. I would rather add a new option here rather than bloat the list of rules

**Comment by ikokostya** [comment]
Close issue, because related PR https://github.com/palantir/tslint/pull/3624 was merged.
